src/foam/graph/GraphBuilder.js

```diff
@@ -39,7 +39,9 @@ foam.CLASS({
 
       // Iterate over rootObject's children
       var parent = this.data[rootObject.id];
-      return rootObject[relationshipKey].dao
+      var relationshipDAO = rootObject[relationshipKey].dao || rootObject[relationshipKey];
```

Comment:
Can `rootObject[relationshipKey]` be undefined?

Reply:
It's  possible but would be odd since the whoever is using the GraphBuilder passes in both the object and the relationshipKey, so if they entered a wrong relationshipKey  then you're right it would break. Ill add logging.